bridge/client/app/_utils/app.utils.ts

```diff
@@ -19,12 +19,13 @@ export class AppUtils {
   }
 
   public static formatNumber(value: number): number {
+    const abs = Math.abs(value);
```

Comment:
This would change the return value from a negative to a positive number.
Suggestion:
```ts
return n * (value < 0 ? -1 : 1)
```

Reply:
In this case it is not, since the `abs` value is only used to decide how many decimals are displayed, but for the calculation itself the original `value` is used. I also added some unit tests to show this and also changed from `Math.floor` to `Math.trunc`, as we have seen with the score, that this makes more sense.